### PR TITLE
Added Windows support

### PR DESCRIPTION
**Pull request by jcoady9**
Added Windows support by creating a new variable in settings.py for the 'sox' command passed to subprocess. Also added links to README for dependencies.

### CODE REVIEW

**Comment by areski** [comment]
Great pullrequest, thank you!
There is one thing, it would be quite important to be fully backward compatible, so people using the actual package won't have anything that break after the upgrade.
Therefore I would strongly recommend we put a default value if `SOX_CLI_COMMAND` is not set.

**Comment by jcoady9** [comment]
Good point, I'll go ahead and set it to 'soxi' by default. Thanks for your consideration!

File: audiofield/fields.py [hunk 1]
```diff
@@ -135,9 +135,14 @@ def _convert_audio(self, filename, instance=None, ext=None):
         filename_temp = filename[:-4] + '_temp'
 
         # Find the number of channels
-        if os.path.isfile(filename):
-            command = "soxi -c %s" % filename
-            response = subprocess.Popen(command.split(' '), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if os.path.isfile(filename): 
+            command = settings.SOX_CLI_COMMAND + "%s" % filename
```
Comment (areski) [inline audiofield/fields.py]: check if SOX_CLI_COMMAND exist and it's defined if not fallback on a default value

**Comment by areski** [comment]
The default should not be in the documentation but in the code so that is the setting is not defined we stay backward compatible. I added a comment in the code where it should be suitable

**Comment by areski** [comment]
Closing this one for now, please reopen if you wish to progress on this

**Pull request closed without merging**